tests/core/pyspec/eth2spec/test/helpers/fork_choice.py

```diff
@@ -60,7 +60,8 @@ def tick_and_add_block(spec, store, signed_block, test_steps, valid=True,
     block_time = pre_state.genesis_time + signed_block.message.slot * spec.config.SECONDS_PER_SLOT
     while store.time < block_time:
         time = pre_state.genesis_time + (spec.get_current_slot(store) + 1) * spec.config.SECONDS_PER_SLOT
-        on_tick_and_append_step(spec, store, time, test_steps)
+        if time > store.time:
```

Comment:
The way the `time` is computed in the above line guarantees that this check never fails